alacritty/src/renderer/box_drawing.rs

```diff
@@ -0,0 +1,584 @@
+            canvas.draw_h_line(bot_right_x, h_lines.1, right_size, stroke_size);
+
+            // Top vertical line.
+            canvas.draw_v_line(v_lines.0, 0., left_top_size, stroke_size);
+            canvas.draw_v_line(v_lines.1, 0., right_top_size, stroke_size);
+
+            // Bottom vertical line.
+            canvas.draw_v_line(v_lines.0, left_bot_y, bottom_size, stroke_size);
+            canvas.draw_v_line(v_lines.1, right_bot_y, bottom_size, stroke_size);
+        },
+        // Arcs: '╭', '╮', '╯', '╰'.
+        '\u{256d}' | '\u{256e}' | '\u{256f}' | '\u{2570}' => {
+            canvas.draw_arc_centered(stroke_size);
+            // Mirror `X` axis.
+            if character == '\u{256d}' || character == '\u{2570}' {
+                let center = canvas.center_h() as usize;
+                for y in 1..height {
+                    let left = (y - 1) * width;
+                    let right = y * width - 1;
+                    for offset in 0..center {
+                        canvas.buffer_mut().swap(left + offset, right - offset);
+                    }
+                }
+            }
+            // Mirror `Y` axis.
+            if character == '\u{256d}' || character == '\u{256e}' {
+                let center = canvas.center_v() as usize;
+                for offset in 1..=center {
+                    let top_row = (offset - 1) * width;
+                    let bottom_row = (height - offset) * width;
+                    for index in 0..width {
+                        canvas.buffer_mut().swap(top_row + index, bottom_row + index);
+                    }
+                }
+            }
+        },
+        // Diagonals: '╱', '╲', '╳'.
+        '\u{2571}' | '\u{2572}' | '\u{2573}' => {
+            let width = width as f32;
+            let height = height as f32;
+            for stroke_size in 0..=stroke_size {
+                let stroke_size = stroke_size as f32 / 2.;
+                if character == '\u{2571}' || character == '\u{2573}' {
+                    canvas.draw_line(stroke_size, height - 1., width - 1., stroke_size);
+                    canvas.draw_line(0., height - 1. - stroke_size, width - 1. - stroke_size, 0.);
+                }
+                if character == '\u{2572}' || character == '\u{2573}' {
+                    canvas.draw_line(stroke_size, 0., width - 1., height - 1. - stroke_size);
+                    canvas.draw_line(0., stroke_size, width - 1. - stroke_size, height - 1.);
+                }
+            }
+        },
+        // Parts of full block: '▀', '▁', '▂', '▃', '▄', '▅', '▆', '▇', '▔', '▉', '▊', '▋', '▌',
+        // '▍', '▎', '▏', '▐', '▕'.
+        '\u{2580}'..='\u{2587}' | '\u{2589}'..='\u{2590}' | '\u{2594}' | '\u{2595}' => {
+            let width = width as f32;
+            let height = height as f32;
+            let rect_width = match character {
+                '\u{2589}' => width * 7. / 8.,
+                '\u{258a}' => width * 3. / 4.,
```

Comment:
Does clippy complain about 4./8. here? If not could we please use `/ 8.` for all of them? My head hurts when I have to do math so just counting from 1 to 7 seems much more pleasant to me. :)

Same goes for other places this is done.

Reply:
The glyphs are named the way you should divide them, but ok, can update to / 8.